components/pedigree/resources/src/main/resources/pedigree/editor.css

```diff
@@ -606,6 +606,28 @@
   width: 48%;
   float: right;
 }
+.menu-box .field-deceasedAge {
+  width: 47%;
+  float: left;
+}
+.menu-box .field-deceasedCause {
+  width: 48%;
+  float: right;
+}
+.menu-box .field-deceasedAge {
+  padding-top: 0;
+  padding-bottom: 0.8em;
+}
+.menu-box .field-deceasedCause {
+  padding-top: 0;
+  padding-bottom: 0.8em;
+}
```

Comment:
afair, @allasm pointed that it is easier to maintain classes if they are unmerged even if it implies repetitions.